pkg/command/cluster/clusteradd/cmd.go

```diff
@@ -169,3 +184,7 @@ func clusterAddedMessage(w io.Writer, id, name string) error {
 const clusterAddNoAuthTokenWarning = `
 WARNING! Scylla data may be exposed
 Protect it by specifying auth_token in /etc/scylla-manager-agent/scylla-manager-agent.yaml on Scylla nodes`
+
+const clusterForceTLSDisabledWarning = `WARNING! TLS is explicitly disabled on cluster sessions (--force-tls-disabled=false).`
```

Comment:
Nice warnings! But I think that you meant to write `(--force-tls-disabled=true)`.